Scripts/Editor/Build/AndroidKeystoreHelper.cs

```diff
@@ -0,0 +1,208 @@
+using Anvil.CSharp.Logging;
+using Anvil.Unity.Editor.UI;
+using UnityEditor;
+using UnityEditor.Build;
+using UnityEditor.Build.Reporting;
+
+namespace Anvil.Unity.Editor.Build
+{
+
+#if UNITY_ANDROID
+    /// <summary>
+    /// Caches keystore information to <see cref="EditorPrefs"/> to that developers don't have to re-enter it each time
+    /// Unity is launched.
+    /// </summary>
+    public class AndroidKeystoreHelper : IPreprocessBuildWithReport
```

Comment:
I don't think that's a good idea. Then you're storing a password in the repo. If the source get stolen the thief can sign binaries under your identity.

Reply:
We don't store the keystore in the repo. We just have it at an agreed upon location that Unity looks for it. 

Same thing with this file.

Reply:
Storing the password in plain text alongside the keystore sort of defeats the purpose of the password no?

Also, a magic file along side the keystore seems more trouble than having developers enter the passwords one time to setup their editor for a project.

Either way, that feature is out of scope for this PR. If someone wants to tackle that we can definitely discuss before work starts.

Reply:
I mean, everyone's just storing the passwords in plain text on their machines now anyway. I agree you don't want that stored in the repo, but you need to enter it which means you need it in some plain text form. Serialized in EditorPrefs is also plain text. 

The benefit of the file is that the devs don't have to do that one time setup. It just exists as part of the process of getting the keystore to the right location anyway.

I agree all of this is out of scope, this solves the major pain point. 

A full solution would be that the keystore and passwords are stored in the repo but encrypted and devs just have to enter the decryption key to this editor tool that will handle decrypting and automatically setting up passwords etc.

Reply:
> I mean, everyone's just storing the passwords in plain text on their machines now anyway.

I would have hoped they were storing their passwords in a password manager.

> Serialized in EditorPrefs is also plain text.

True but it's the difference between taping the extra set of keys for your house to your front door (file) vs putting them under a super obvious rock (EditorPrefs). 😆 

> The benefit of the file is that the devs don't have to do that one time setup. It just exists as part of the process of getting the keystore to the right location anyway.

Yea, I guess if the helper supported navigating to the special directory and validating its contents it wouldn't be so bad but otherwise I don't love the idea of having a magic file name, location and format to support this. It's a very opaque requirement.